ycms/cms/templates/zentral.html

```diff
@@ -0,0 +1,86 @@
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "YCMS at a glance" %}</div>
+            <div class="grid grid-cols-3">
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Wards" %}</div>
+                    <div class="text-3xl font-bold">{{ wardscount }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Available Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ availablebeds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Doctors" %}</div>
+                    <div class="text-3xl font-bold">{{ doctors }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ beds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Occupied Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ occupiedbeds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Nurses" %}</div>
+                    <div class="text-3xl font-bold">{{ nurses }}</div>
+                </div>
+                <div class="col-span-2 sm:col-span-1">
+                    <div class="text-lg font-semibold">{% translate "Ward Details" %}</div>
+                    <div class="text-2xl">
+                        <ul>
+                            {% for ward in wards %}<li>{{ ward }}</li>{% endfor %}
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Add a New Ward" %}</div>
+            <br>
+            <form enctype="multipart/form-data"
+                  method="post"
+                  id="ward_form"
+                  class="mb-12"
+                  data-unsaved-warning>
+                {% csrf_token %}
+                {{ wardform }}
+                <br>
+                <button type="submit" class="btn mt-4">{% translate "Add New Ward" %}</button>
+            </form>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Wards Detail" %}</div>
```

Comment:
```suggestion
            <div class="mb-4 text-2xl font-semibold">{% translate "All Wards" %}</div>
```
Super pendantic, and feel free to ignore. At least to me (and I may be wrong) "Wards Detail" reads more like "a details about all the wards" than "details about each ward".